### DISCUSSION TITLE
In which case the pages/error.vue from Frontends-Demo is used?

### DISCUSSION THREAD

**Discussion by realconvis** [shopware/frontends#377] [Q&A] [answered]
I wonder in which case the pages/error.vue from Frontends-Demo is used?

If [...all].vue component didn´t find a route, the /components/errors/RoutingNotFound.vue is used.
According the nuxt documentation a error.vue has to be place in root folder but with diffrent props.

So i can´t find out, in which case the pages/error.vue is used.
Someone can help?

**Comment by mdanilowicz** [comment, marked as answer]
Hi, @realconvis 

Yes, the place of the `error.vue` file was wrong. I fixed it here https://github.com/shopware/frontends/pull/388. Thank you for your reporting. 

Backing to the main topic. Right now, it works like this: 
- `RoutingNotFound` will be displayed when [..all].vue will not be able to find any routing
- Error page will be displayed for "heavier" errors like 500 errors from the node server, etc. 

On the error page, we are not displaying the menu, footer, etc. This is just a page with error info.
